src/Uno.UI/UI/Xaml/Controls/ToggleSwitch.cs

```diff
@@ -60,11 +56,9 @@ protected override void OnUnloaded()
 
 			if (!IsNativeTemplate)
 			{
-				RemoveHandler(PointerPressedEvent, (PointerEventHandler) OnPointerPressed);
-				RemoveHandler(PointerExitedEvent, (PointerEventHandler) OnPointerExited);
-				RemoveHandler(PointerReleasedEvent, (PointerEventHandler) OnPointerReleased);
-				RemoveHandler(PointerCanceledEvent, (PointerEventHandler) OnPointerCanceled);
-				RemoveHandler(PointerEnteredEvent, (PointerEventHandler) OnPointerEntered);
+				RemoveHandler(PointerPressedEvent, (PointerEventHandler)OnPointerPressed);
+				RemoveHandler(PointerReleasedEvent, (PointerEventHandler)OnPointerReleased);
+				RemoveHandler(PointerCanceledEvent, (PointerEventHandler)OnPointerCanceled);
```

Comment:
No more Entered/Exited ?

Reply:
No, there were used only for visual states, which are now handled by the mixins